docs/setup_lnd.md

```diff
@@ -17,7 +17,7 @@ Clone the peerswap repository and build the peerswap plugin
 ```bash
 git clone https://github.com/ElementsProject/peerswap.git && \
 cd peerswap && \
-make lnd-install
+make lnd-release
 ```
 
 This will install `peerswapd` and `pscli` to your go path
```

Comment:
The docs state that the files will be installed into the users go path, but looking at the makefile this does not seem to be true. The files are just built in the peerswap root directory. So maybe also change this line to match what this command really does, and/or add a small info on how to move the files to the GOPATH?

Reply:
Can confirm that `peerswapd` and `pscli` is installed in the current `peerswap` directory.
Instead of the GOPATH I move them to a directory in the PATH like `/usr/local/bin/`